modules/services/grobi.nix

```
@@ -0,0 +1,107 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.grobi;
+  eitherStrBoolIntList = with types; either str (either bool (either int (listOf str)));
+
+in
+
+{
+  meta.maintainers = [ maintainers.mbrgm ];
+
+  options = {
+    services.grobi = {
+
+      enable = mkEnableOption "the grobi display setup daemon";
+
+      executeAfter = mkOption {
+        type = with types; listOf str;
+        default = [];
+        description = ''
+          Commands to be run after an output configuration was changed. The Nix
+          value declared here will be translated to JSON and written to the
+          `execute_after` key in ~/.config/grobi.conf.
```

Need docbook formatting
```suggestion
          <option>execute_after<option> key in <filename>~/.config/grobi.conf</filename>.
```